src/Illuminate/Collections/Arr.php

```diff
@@ -702,11 +702,12 @@ public static function pull(&$array, $key, $default = null)
      * Convert the array into a query string.
      *
      * @param  array  $array
+     * @param  int-mask-of<PHP_QUERY_*> $encodingType (optional) PHP Query encoding type.
```

Comment:
Analogously to `Request::fullUrlWithoutQuery()` as requested in https://github.com/laravel/framework/pull/54464#discussion_r1941392172 and changed in 52b2123, the language would have to be removed here as well:
```suggestion
     * @param  int-mask-of<PHP_QUERY_*> $encodingType (optional) Query encoding type.
```